input/test1.asm

```diff
@@ -1,24 +1,18 @@
-org 0
-jmp start
-
-valor1 db 5
-valor2 db 7
-mayor  db 0
-const ten 0A
-
-AnotherOne:
-    ADD R1, R3
-    JMP fin
-
+    org 0
+    JMPADDR start
+	org 4
+valor1 db 8
+const pi 31
+	org 0E
 start:
-    LOAD R1, valor1
-    LOAD R2, valor2
-    GRT  R1, R2
-    JMP AnotherOne
-
-R1esMayor:
-    STORE mayor, R1
-    LOADIM R3, #8
-
-fin:
-    JMP fin
\ No newline at end of file
+	LOAD R1, valor1
+	LOADIM R3, pi
+	GRT R1, R2
+	JCONDADDR aca
+	LOADIM R4, 0F
+stay1:
+	JMPADDR stay1
+aca:
+	LOADIM R4, 0A
+stay2:
+	JMPADDR stay2
```

Comment:
Revert entire file. This destroys previously implemented testers in tests directory

Reply:
Only the first of the previous implemented testers was fully functional. The rest were just to check for errors, which were appropriately addressed. These new testers are the official ones the client gaves us, which means we should use them to fix the bugs on the assembler. Even so, I still left intact the first working tester that was originally made.